packages/base/zarf.yaml

```diff
@@ -10,6 +10,11 @@ metadata:
   version: "0.39.0"
   # x-release-please-end
   x-uds-dependencies: []
+  annotations:
+    dev.uds.title: UDS Core (Base)
+    dev.uds.tagline: UDS Core mTLS, zero trust, and Pepr
```

Comment:
Can we update taglines to the descriptions from docs.  https://uds.defenseunicorns.com/reference/uds-core/functional-layers/

Reply:
Thank you, Updated!